examples/eestream/serve-collected/main.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"crypto/sha256"
+	"flag"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/vivint/infectious"
+	"storj.io/storj/pkg/eestream"
+	"storj.io/storj/pkg/ranger"
+)
+
+var (
+	addr           = flag.String("addr", "localhost:8080", "address to serve from")
+	pieceBlockSize = flag.Int("piece_block_size", 4*1024, "block size of pieces")
+	key            = flag.String("key", "a key", "the secret key")
+	rsk            = flag.Int("required", 20, "rs required")
+	rsn            = flag.Int("total", 40, "rs total")
+)
+
+func main() {
+	err := Main()
+	if err != nil {
+		panic(err)
```

Comment:
I don't think we should panic here. os.Exit should work with a log line

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#dont-panic

Reply:
this was my bad

Reply:
I promise to never panic again and always have my towel with me.